.golangci.yml

```diff
@@ -95,7 +95,7 @@ linters-settings:
     - name: unused-parameter
       disabled: true
   sloglint:
-    context-only: true
+    context: all
```

Comment:
"context: all" is the same as "context-only: true".

There is a more lenient setting now, "context: scope". Arguably we could move to it, although I've elected not for the time being (it's easier to relax a linter than to tighten it).

https://golangci-lint.run/usage/linters/#sloglint